### ISSUE TITLE
Keydown event for readOnly cells not propagate to the higher-level elements 

### ISSUE THREAD

**Issue by OlgaIOn** [handsontable/handsontable#7643]
### Description
We want to deal with keydown events, fired on hot-cells, in higher-level elements. This works for the editable cells, but does not work if the cell is readOnly.

### Steps to reproduce

1. Select cell, press any key -> console: EventListener 'keydown' was called 
2. Make cell readonly in context menu, select cell, press any key -> console: EventListener 'keydown' was not called 

### Demo
 https://jsfiddle.net/o40ew5jq/

### Your environment
* Handsontable version: 7.2.2

**Comment by AMBudnik** [comment]
Hi @OlgaIOn 

I see that you have also placed our official hooks
* `beforeKeyDown`
* `afterDocumentKeyDown `

and those work as expected. So I would like to ask you for the requirements. Why there's a need to use a global hook if Hadsontable allows you to track every event with the supported official API.

**Comment by OlgaIOn** [comment]
Hello @AMBudnik 

We implement hotkeys in our application and want the elements outside of the HoT to be aware of the key combinations pressed on selected HoT-cell, so that outside elements can react too. 

It's about the logic for the external for HoT components, the HoT hooks are hier not sutable. 

We wonder why the keypress events for editable cells are passed to the upper elements and for the non-editable cells not. 

Could you change this?

**Comment by AMBudnik** [comment]
Within the description of `afterDocumentKeyDown` we can read that

![image](https://user-images.githubusercontent.com/10757813/112112041-4c471380-8bb5-11eb-9c94-befd94c7cde9.png)

And there is no API to change that behavior. 

I have created a sample demo here https://jsfiddle.net/handsoncode/b5fu2o0w/ where I set up a function that gathers the `event.type` and `event.key` and runs for the container as in your demo. But it also runs the function in the `afterDocumentKeyDown` hook for the cells that are `readOnly`. Is that getting a bit closer to the project requirements that you have?
